lib/activity/router.js

```diff
@@ -57,13 +57,13 @@ module.exports = function route(callback) {
                 channelId: subscription.channelId,
                 creatorId: subscription.creatorId,
                 githubId: subscription.githubId,
+                githubName: subscription.githubName,
                 workspaceId: subscription.SlackWorkspace.slackId,
               },
             }, 'User lost access to resource. Deleting subscription.');
 
             await Promise.all([
-              // @todo: deactive this subscription instead of deleting the db record
-              await subscription.destroy(),
+              await subscription.destroyWithReason('lost access'),
```

Comment:
could we update this to `subscription creator lost access`?

```suggestion
              await subscription.destroyWithReason('subscription creator lost access'),
```